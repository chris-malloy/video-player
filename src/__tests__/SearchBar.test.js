import React from 'react';
import SearchBar from '../components/SearchBar';
import renderer from 'react-test-renderer';

describe('Component: SearchBar', () =>{

    test('SearchBar renders correctly', () => {
        const component = renderer.create(
            <SearchBar input={'input'}>SearchBar</SearchBar>
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});

