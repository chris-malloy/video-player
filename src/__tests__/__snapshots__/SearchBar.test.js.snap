// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: SearchBar SearchBar renders correctly 1`] = `
<div>
  <input
    onChange={[Function]}
    value=""
  />
</div>
`;
